tests/Unit/Availability/AvailabilityTestHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Unit\Availability;
+
+use App\User;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+trait AvailabilityTestHelper
```

Comment:
I think this is a copy of the same helper for bookings? Feel free to put them in into one helper

Reply:
Done